Periodic_2_triangulation_2/include/CGAL/Periodic_2_triangulation_2.h

```diff
@@ -1424,6 +1384,10 @@ class Periodic_2_triangulation_2: public Triangulation_cw_ccw_2
   /// The domain
   Iso_rectangle _domain;
 
+protected:
+  // @fixme this covering stuff should really be at the Delaunay level (will need
```

Comment:
is this remark for later or did you forget it?

Reply:
It's a remark left for someday.

These are all members that should be in the P2DT2 class and not P2T2. Because there is no P2Regular_triangulation_2, it does not matter right now. I didn't move these members because too much code needed to be converted to a generic form (like it is done in P3T3).

Reply:
OK.